test/jestUtils.js

```diff
@@ -0,0 +1,19 @@
+const isDeprecatedLifecycleWarning = (msg, componentName) => {
+  return (
+    msg.includes('has been renamed, and is not recommended for use') &&
+    msg.includes(componentName)
```

Comment:
Do we know if we have these warnings because we didn't migrate to UNSAFE_ lifecycle methods, or if one of our dependencies didn't do it yet?

Reply:
When using react-redux 5's `connect`, the wrapped component will issue this warning. The jest.setup.js details the "reason"s for each ignored warning.

Reply:
OK thanks